### ISSUE TITLE
How to use mask guidance？

### ISSUE THREAD

**Issue by GuangSTrip** [TinyBeauty/TinyBeauty#7]
Hello, I would like to inquire about how to use the mask guidance mentioned in the paper in the fine-tuning process.

**Comment by Au233** [comment]
In training process, we only focus the learning on M_changed regions to effectively capture makeup style. This is achieved by computing the loss Lsimple only over the features within the M_changed area, you can find corresponding code in https://github.com/TinyBeauty/TinyBeauty/blob/f42463518465548d8bde003d6da30c34c7f9d430/SD_finetune.py#L635

**Comment by GuangSTrip** [comment]
Thank you for your reply! And I have learned how to use mask guidance after delving deeper into SD_finetune. py and SD_inference. py
